.travis.yml

```diff
@@ -9,7 +9,7 @@ git:
 os:
   - linux
 env:
-  - V=0.23.0
+  - V=0.28.0
```

Comment:
Could we support at least two latest versions? `0.27.2` and `0.28.1` at this moment.

Reply:
We don't need to support multiple Bazel versions for this because the consumable is pre-built.